app/forms/school_cohort_form.rb

```diff
@@ -7,4 +7,6 @@ class SchoolCohortForm
 
   validates :estimated_mentor_count, presence: { message: "Enter your expected number of mentors" }
   validates :estimated_teacher_count, presence: { message: "Enter your expected number of ECTs" }
+  validates :estimated_mentor_count, numericality: { greater_than_or_equal_to: 0 }
+  validates :estimated_teacher_count, numericality: { greater_than_or_equal_to: 0 }
```

Comment:
We could do it in the same lines I think.
```
validates :estimated_mentor_count, presence: { message: "Enter your expected number of mentors" }, numericality: { greater_than_or_equal_to: 0 }
```

Unless you think it makes things less readable of course.

Reply:
I think it's better separated. Shorter, so I don't need to worry about splitting to multiple lines and easier to read too